### ISSUE TITLE
PreferenceContentText is forced to TextAlign.End even when using contentPlacementBottom = true

### ISSUE THREAD

**Issue by jscti** [MFlisar/ComposePreferences#2]
Hi there

Small bug when using (on a PreferenceListMulti)  :  
```
itemSetup = PreferenceListDefaults.itemSetup().copy(
    contentPlacementBottom = true,
), 
```

Then text is aligned to the end (when text is quite long and wraped to next line) because it is hardcoded in the `PreferenceContentText` composable :  

![image](https://github.com/MFlisar/ComposePreferences/assets/1594035/9d719c85-0cc2-459f-a085-b81e0051c43b)

**Comment by MFlisar** [comment]
Solved in [Release 0.4.1](https://github.com/MFlisar/ComposePreferences/releases/tag/0.4.1)

**Comment by jscti** [comment]
Thanks ! That was quick !
